www/js/controllers.js

```diff
@@ -1,38 +1,51 @@
-angular.module('starter.controllers', [])
+angular.module('starter.controllers', ['ionic'])
 
-.controller('AppCtrl', function($scope, $ionicModal, $timeout, $auth, $window, $http, $state) {
+.controller('AppCtrl', function(CtrlService, $scope, $auth, $window, $http, $state) {
   // controls the display of hamburger navicon
-
   var validateUser = function(){
-    $scope.currentUser = JSON.parse($window.localStorage.getItem('current-user'))
-      console.log($scope.currentUser.uid + ' is logged in.');
+    $scope.currentUser = CtrlService.getUser(); //set value to show hamburger menu
+    console.log($scope.currentUser);
+    // console.log('CURRENT USER---->', $scope.currentUser);
       if ($scope.currentUser != null){
-        $state.go('app.home')
+        $state.go('app.home');
+      } else{
+        $state.go('app.landing');
       }
   }
 
-  validateUser(); //gets the current user
-
-})
+  $scope.$watch(validateUser()); 
+  $scope.$watch($scope.currentUser); 
 
-.controller('landingCtrl', function($scope, $auth, $http, $window, $state) {
-  $scope.signinForm = {};
+  $scope.signout = function(){
+    CtrlService.clearUser();
+    $scope.$apply;
```

Comment:
@ggharry This is for the signup page if the user has not signed in. If localStorage has no current User stored